content/docs/studio/user-guide/model-registry/assign-stage.md

```diff
@@ -1,18 +1,29 @@
-# Assign stages to models
+# Assign stage to model version
```

Comment:
Are there any links to this section from anywhere else in the docs/studio? If yes, they'll need to be updated.

Reply:
The page URL doesn't change, so unless I'm missing something, nothing breaks.

Reply:
If a page has multiple sections, sometimes we also use the section url specifically (such as `doc/studio/user-guide/model-registry/register-version#version-models`). Maybe there's none for this section, but would be good to confirm.

Reply:
Right, thanks. Confirmed!